src/3rd_party/pathie-cpp/src/pathie.cpp

```diff
@@ -206,11 +205,10 @@ std::string Pathie::utf8_to_filename(const std::string& utf8)
 std::string Pathie::filename_to_utf8(const std::string& native_filename)
 {
   bool fs_encoding_is_utf8 = false;
-
+  char* fsencoding = NULL;
```

Comment:
same question here--why moved up? If not needed, please could you move it back; the original seemed more local.

Reply:
Same as the above, it's defined inside of the `else` branch of an `ifdef` so the code doesn't compile if the preprocessor takes the other branch.

Reply:
Thanks